lib/plausible_web/controllers/stats_controller.ex

```diff
@@ -90,6 +92,18 @@ defmodule PlausibleWeb.StatsController do
     end
   end
 
+  on_full_build do
+    def list_funnels(site) do
+      Plausible.Funnels.list(site)
+    end
+  end
+
+  on_small_build do
+    def list_funnels(_site) do
+      []
+    end
+  end
+
```

Comment:
```suggestion
  on_full_build do
    defp list_funnels(site) do
      Plausible.Funnels.list(site)
    end
  end

  on_small_build do
    defp list_funnels(_site) do
      []
    end
  end

```

or do we need them to be explicitly public?